src/send/mod.rs

```diff
@@ -0,0 +1,99 @@
+#[cfg(test)]
+mod test;
+
+use rocket::{
```

Comment:
Good practice to arrange these according to this Style guide: https://doc.rust-lang.org/1.0.0/style/style/imports.html

Reply:
and nitpick: Personally I'd stick w/ the default cargo fmt options, I rarely see all of these :)